README.md

```diff
@@ -22,6 +22,20 @@ docker compose exec django python manage.py init
 # Wait a little, then check that https://localhost/oapif/collections/signalo_core.pole/items works from your browser
 ```
 
+## Authentication & permissions
+
+By default the viewsets under `signalo/core` use the `AllowAny` permissions class. You can override these app-level permissions for particular viewsets when registering their corresponding models. (Refer to https://www.django-rest-framework.org/api-guide/permissions/#api-reference for permission classes). Example:
+
+    models.py
+    ---------
+    from rest_framework.permissions import DjangoModelPermissionsOrAnonReadOnly
+    ...
+    @register_oapif_viewset(
+        custom_viewset_attrs={
+            "permissions_classes": (DjangoModelPermissionsOrAnonReadOnly,)
+        }
+    )
```

Comment:
as said, please include a Model definition here, otherwise the decorator is hanging and it's not clear on what it applies